src/components/calcite-tile-select/calcite-tile-select.scss

```diff
@@ -62,14 +62,14 @@ $spacing: $baseline/2;
 
 :host([heading]:not([icon]):not([description])) {
   .root {
-    align-items: center;
+    @apply items-center;
   }
 }
 
 :host([input-enabled][input-alignment="start"][icon][heading][description]),
 :host([input-enabled][input-alignment="start"]:not([icon])[heading]:not([description])) {
   .root {
-    display: inline-grid;
+    @apply inline-grid;
     grid-gap: $spacing;
     grid-template-columns: max-content 1fr;
```

Comment:
Did you consider refactoring to use the `grid-col-` util?

Reply:
How _would_ you use any tailwind grid helpers for something like this?  Grid stuff is usually very specific and the generalized tailwind classes don't usually accommodate the need there.